documentation/REFERENCE.md

```diff
@@ -141,3 +141,30 @@ scrollphathd.flip(x=False, y=False)
 Parameters:  
 x – Flip horizontally left to right  
 y – Flip vertically up to down  
+
+## Run a http API
+
+flask must be installed before the api bluerint can be imported. 
+To build up a flask api using the blueprint you can use the following and add your own blueprints to the api as you choose:
+```python
+from scrollphathd import scrollphathd_blueprint
+from flask import Flask
+
+
+app = Flask(__name__)
+app.register_blueprint(scrollphathd_blueprint, url_prefix='scrollphathd')
```

Comment:
If I don't prefix the `url_prefix=''` value with a / I get the error:

```
Traceback (most recent call last):
  File "/usr/local/bin/scrollphathd_api", line 11, in <module>
    load_entry_point('scrollphathd', 'console_scripts', 'scrollphathd_api')()
  File "/home/pi/Development/scroll-phat-hd/library/scrollphathd/api/http.py", line 127, in main
    app.register_blueprint(scrollphathd_blueprint, url_prefix="scrollphathd")
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 62, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 889, in register_blueprint
    blueprint.register(self, options, first_registration)
  File "/usr/lib/python2.7/dist-packages/flask/blueprints.py", line 153, in register
    deferred(state)
  File "/usr/lib/python2.7/dist-packages/flask/blueprints.py", line 172, in <lambda>
    s.add_url_rule(rule, endpoint, view_func, **options))
  File "/usr/lib/python2.7/dist-packages/flask/blueprints.py", line 76, in add_url_rule
    view_func, defaults=defaults, **options)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 62, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 976, in add_url_rule
    rule = self.url_rule_class(rule, methods=methods, **options)
  File "/usr/lib/python2.7/dist-packages/werkzeug/routing.py", line 537, in __init__
    raise ValueError('urls must start with a leading slash')
ValueError: urls must start with a leading slash
```
Running Flask 0.10.1 in Python 2.7.9